docker-compose.yml

```diff
@@ -32,6 +32,7 @@ services:
       LOGLEVEL: "DEBUG"
       ENABLE_GRAPHQL: 1
       NODE_ENV: "development"
+      EXTERNAL_AUTH_JWKS_URLS: "https://cognito-idp.us-east-1.amazonaws.com/us-east-1_e4iVm1J4X/.well-known/jwks.json"
 #      SLACK_TOKEN: ""
```

Comment:
can this be a comma separated list?

Reply:
@imranq2 yes! that was the intention that eventually this will be a comma separated string as we get more IdPs connected